tutorials/vr/developing_for_oculus_quest.rst

```diff
@@ -0,0 +1,126 @@
+.. _doc_developing_for_oculus_quest:
+
+Introduction
+------------
+
+This tutorial goes over how to get started developing for the
+Oculus Quest with an official plugin.
+
+First there are two things you need to do.
+
+First you need to go through the steps on the :ref:`doc_exporting_for_android`
```

Comment:
You are repeating yourself with "First"

I would skip this line completely `First there are two things you need to do.` it's a filler and brings nothing new to the table.